### ISSUE TITLE
Draggable views row weights not dispalying/usable

### ISSUE THREAD

**Issue by akslay** [az-digital/az_quickstart#2031]
## Problem/Motivation
The row weights field for draggable views (i.e., "Reorder People") is not displaying row weight values, which makes them unusable for site editors. This is also an accessibility problem.

Example: https://quickstart.arizona.edu/admin/az-person/reorder-people
![Screen Shot 2022-11-01 at 1 45 25 PM](https://user-images.githubusercontent.com/52931751/199337435-49128690-9793-474a-ba23-5296896f15fc.png)

### Related Drupal Issues
- https://www.drupal.org/project/draggableviews/issues/3252365
- https://www.drupal.org/project/draggableviews/issues/3229357
- https://www.drupal.org/project/draggableviews/issues/2929116
- https://www.drupal.org/project/draggableviews/issues/3270935

## Proposed resolution
Contribute a patch to the contrib module or add a bandaid solution to Quickstart until the contrib module is fixed.

### Expected behavior
Editors should be able to use row weights instead of draggable handles to reorder nodes.

**Comment by akslay** [comment]
Appears the issue may have been re-introduced after switching to 2.0.1 module version? Other users on Drupal.org report that switching from 2.x version back to 8.x version solves the problem for them.

**Comment by akslay** [comment]
Seems more like a CSS issue, though, since the input fields are rendering and have values stored in them. Applying a hard-coded width does allow the values to be seen. Checking to see if this is another "fork Barrio" issue.

**Comment by akslay** [comment]
![Screen Shot 2022-11-01 at 2 15 10 PM](https://user-images.githubusercontent.com/52931751/199343055-1dff64e3-9ae1-45b6-9dcd-cf4c0b22c7b4.png)
Looks like it's a Claro theme problem (Drupal core problem).

- https://www.drupal.org/project/drupal/issues/3135457
